srcpkgs/font-Siji/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'font-Siji'
+pkgname=font-Siji
+version=0.0.180725
+revision=1
+build_style=fetch
+hostmakedepends="git"
+short_desc="An iconic bitmap font based on Stlarch with additional glyphs"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="GPL-2.0"
+homepage="https://github.com/stark/siji"
+font_dirs="/usr/share/fonts/misc"
+
+do_fetch() {
+	git clone "${homepage}.git" "${wrksrc}"
```

Comment:
please prefer fetching a tarball of the commit, see `compton` template for an example